contracts/staking/StakingRewardsUpgradeable.sol

```diff
@@ -31,6 +31,7 @@ contract StakingRewardsUpgradeable is
     uint256 public periodFinish;
     uint256 public override rewardRate;
     uint256 public override rewardsDuration;
+    uint256 public override minimumStakeAmount;
```

Comment:
New state variables must be added after any and all existing state variables, for proxy contract upgradeability reasons,